src/ServiceStack.OrmLite.Oracle.Tests/OracleParamTests.cs

```diff
@@ -269,328 +259,922 @@ public void ORA_ParamByteTest()
         }
 
         [Test]
-        public void ORA_ReservedNameComment_Test()
+        public void ORA_ReservedNameLevelAsAlias_Test()
         {
             using (var db = OpenDbConnection())
             {
                 DropAndCreateTables(db);
 
-                var row = new ParamComment { Comment = 1, Id = 2 };
+                var row = new ParamLevelAsAlias { Id = 2, ApprovalLevel = 212218 };
                 db.Insert(row);
 
-                row.Comment = 454;
+                row.ApprovalLevel = 676;
                 db.Update(row);
             }
         }
 
+        [ImportMany(typeof(IParam))]
```

Comment:
Yes, I used MEF in the Oracle unit tests to resolve all the parameter classes and cut down on code. In the future if we want to test a new parameter we can just implement another class that inherits from IParam and it will automatically be included in the ORA_ReservedNames_Test.